src/main/java/com/yandex/money/api/model/showcase/components/Component.java

```diff
@@ -8,6 +10,13 @@
  */
 public abstract class Component {
 
+    @Override
+    public String toString() {
+        return getToStringBuilder().toString();
+    }
+
+    protected abstract ToStringBuilder getToStringBuilder();
```

Comment:
The extraction should be performed in another task because it can be lead to one usage interface for a long time. Moreover, method will be `public`.